<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xuanyingxiong</class>
 <widget class="QDialog" name="xuanyingxiong">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选英雄界面</string>
  </property>
  <widget class="QRadioButton" name="radioButton1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>马里奥</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>115</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>喜羊羊</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>光头强</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>冰火人</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>奥特曼</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择英雄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
